<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIgesturerecognitiondtw</class>
 <widget class="QWidget" name="GUIgesturerecognitiondtw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Gesture Recognition with HMM</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3290</width>
     <height>961</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dynamic  Time Warping GUI Unit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="boxWS">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Window Size</string>
    </property>
    <widget class="QLabel" name="labelWS">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>361</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Gesture Labels</string>
    </property>
    <widget class="QLabel" name="labelGestureLabel">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>20</y>
       <width>331</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxDTWvalue">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>DTW value</string>
    </property>
    <widget class="QLabel" name="labelDTWvalue">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>32</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxClass">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>371</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Gestures Classified</string>
    </property>
    <widget class="QLabel" name="labelClass">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>23</y>
       <width>341</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAllScores">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>All DTW values</string>
    </property>
    <widget class="QLabel" name="labelAllScores">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxClassDTW">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Classification</string>
    </property>
    <widget class="QLabel" name="labelClassDTW">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="chart" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>1221</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <action name="actionProject">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionKk">
   <property name="text">
    <string>kk</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
